plotting/timeseries.py

```diff
@@ -72,36 +68,34 @@ def load_data(self):
             else:
                 self.depth_label = ''
 
-            if not (set(dataset.variables[var].dimensions) &
+            # Override depth request if requested variable has no depth (i.e. surface only)
+            if not (set(dataset.variables[variable].dimensions) &
                     set(dataset.nc_data.depth_dimensions)):
                 self.depth = 0
 
-            times = None
             point_data = []
+            dep = []
```

Comment:
I think `dep` should be renamed to `depths` or something similar just for clarity